src/screen/ProductPage.tsx

```diff
@@ -98,13 +144,9 @@ export function ProductPage(props: ProductPageProps) {
           setInputProduct(newCategoryId);
         }}
       >
-        {categories?.map((category) => {
+        {fetchCategories.categories.map((category: Category) => {
```

Comment:
`category` ini harusnya gak perlu dikasih type lagi, soalnya udah dapat infer dari `fetchCategories.categories`

```suggestion
        {fetchCategories.categories.map((category) => {
```